lib/apr_web/live/order_dashboard_live.ex

```diff
@@ -6,11 +6,6 @@ defmodule AprWeb.OrderDashboardLive do
   def render(assigns) do
     ~L"""
     <div class="main-live">
-      <div class="main-controllers">
```

Comment:
removed buttons from the top as we discussed they are not as useful

Reply:
I may and or may not have spent time making them pretty in another PR 😂

Reply:
im sure we'll need buttons again